doc/transform_methods/basex-comp.rst

```diff
@@ -0,0 +1,298 @@
+:orphan:
+
+.. _BASEXcomp:
+
+BASEX: computational details
+============================
+
+The BASEX article does not provide the derivation of the basis projections and
+is very terse regarding their computation, so here we provide the missing
```

Comment:
Can we re-word this to make it a little kinder to the BASEX manuscript. Maybe something like, "Here we provide the full derivation of the basis projections that are summarized in Dribinski et al."

Reply:
They deserve it! :–)
What about “To complement the general description given in the BASEX article, here we
provide the full derivation of the basis projections and the details needed for
their efficient computation.”?

Reply:
> They deserve it! :–)

Ha!

> What about “To complement the general description given in the BASEX article, here we
provide the full derivation of the basis projections and the details needed for
their efficient computation.”?

Sounds good :)